features/support/ui/landing.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'ui'
+
+module UI
+  class Landing < SitePrism::Page
+    set_url '/'
+    set_url_matcher(/wpcc\/\d+\/landing/)
+
+    element :age, '#your_details_form_age'
```

Comment:
I respectfully disagree with the de facto standard which adds classes to the markup to facilitate cucumber specs. Rails automatically adds these ids to the markup,  so I would question if we want to add more bloat.